scout/server/blueprints/cases/controllers.py

```diff
@@ -448,8 +448,12 @@ def update_individuals(store, institute_obj, case_obj, user_obj, ind, age, tissu
     case_individuals = case_obj.get('individuals')
     for subject in case_individuals:
         if subject['individual_id'] == ind:
-            subject['age'] = round(float(age), 1)
-            subject['tissue_type'] = tissue
+            if age:
+                subject['age'] = round(float(age), 1)
+            else:
+                subject['age'] = None
+            if tissue:
```

Comment:
Ah! o_O O_o This was a tricky one to deduce from the server error. Clearly safer now!